bindings/python/fdb/__init__.py

```diff
@@ -100,7 +100,7 @@ def api_version(ver):
 
     _add_symbols(fdb.impl, list)
 
-    if ver >= 710:
+    if ver >= 720:
```

Comment:
Revert. I think this is since 710. @sfc-gh-ajbeamon can you confirm?

Reply:
fixed.